docs/quickstart.rst

```diff
@@ -1,9 +1,27 @@
 Quickstart
 ==========
 
+Using FullNodeClient
+--------------------
+A :ref:`Client` is a facade for interacting with Starknet.
+:ref:`FullNodeClient` is a client which interacts
+with a Starknet full node like `pathfinder <https://github.com/eqlabs/pathfinder>`_.
```

Comment:
Maybe we could mention more nodes?
- https://github.com/starkware-libs/papyrus
- https://github.com/NethermindEth/juno